app/views/works/index.html.erb

```diff
@@ -0,0 +1,70 @@
+
+<main>
+  <h2>All Works</h2>
+  <section class="media-table">
+    <h4>ALBUMS</h4>
+      <table class="table">
+        <tr>
+          <th>Votes</th>
+          <th>Title</th>
+          <th>Created By</th>
+          <th>Published</th>
+          <th>Upvote</th>
+        </tr>
+        <% @albums.each do |work| %>
+            <tr>
+              <td><%= work.votes.count %></td>
+              <td><%= link_to work.title, work_path(work.id) %></td>
+              <td><%= work.creator %></td>
+              <td><%= work.publication_year %></td>
+              <td><%= link_to "VOTE", work_path(work.id), class: "btn btn-primary"%></td>
+            </tr>
+        <% end %>
+      </table>
+  </section>
```

Comment:
These `media-table` sections would be a great candidate for a partial view that took a couple arguments because they're all three almost exactly the same. Sorry I wasn't very clear about explaining that in my previous feedback.